daq/docker_test.py

```diff
@@ -108,26 +112,26 @@ def _check_image(self):
 
     def terminate(self):
         """Forcibly terminate this container"""
-        LOGGER.info("Target port %d test %s terminating", self.target_port, self.test_name)
+        LOGGER.info("%s terminating", self)
```

Comment:
Why is it sometimes "Target device %s" % self.device, and other times just "%s" % self?  Line 104 vs 115... I would expect them to be the same?